interactive.js

```diff
@@ -99,22 +90,20 @@ const preferHeurisicallyInterestingProcesses = (a, b) => {
 };
 
 const filterProcesses = (input, processes, flags) => {
-	const filters = {
-		name: process_ => input ? nameFilter(input, process_) : true,
-		verbose: process_ => input ? (isWindows ? process_.name : process_.cmd).toLowerCase().includes(input.toLowerCase()) : true,
-	};
-
 	const memoryThreshold = flags.verbose ? 0 : 1;
 	const cpuThreshold = flags.verbose ? 0 : 3;
+	const fuzzySearchOption = {caseSensitive: false};
+
+	const filteredProcesses = flags.verbose
+		? new FuzzySearch(processes, [isWindows ? 'name' : 'cmd'], fuzzySearchOption).search(input)
+		: new FuzzySearch(processes, ['name'], fuzzySearchOption).search(input);
```

Comment:
The ternary has a lot of duplicate code. I think you can narrow the scope of the ternary.